React/Modules/RCTAlertManager.m

```diff
@@ -64,7 +64,7 @@ - (void)invalidate
   NSString *message = [RCTConvert NSString:args[@"message"]];
   NSString *type = [RCTConvert NSString:args[@"type"]];
   NSDictionaryArray *buttons = [RCTConvert NSDictionaryArray:args[@"buttons"]];
-  BOOL allowsTextInput = [type isEqual:@"plain-text"];
+  BOOL allowsTextInput = [type length] != 0;
```

Comment:
Maybe get rid of this boolean since it serves little purpose now. Just change the last clause in the multi-way if statement below to `else if (type.length) { ...log error }`